### DISCUSSION TITLE
Sponsored Products Sales Data

### DISCUSSION THREAD

**Discussion by Nilanth** [amzn/ads-advanced-tools-docs#36] [Q&A] [answered]
We are receiving multiple columns for sales value like **sales1d**, **sales7d**, **sales14d**, and **sales30d**, In these columns which holds the sales value or do we need to sum all the columns?  I'm using the `/reporting/reports` API for the **spPurchasedProduct** report. 

Also, **[sales7d](https://advertising.amazon.com/API/docs/en-us/reporting/v3/metrics#sales7d)** states **the total value of sales occurring within 7 days of an ad click** as per the docs. So do we request the same date after 7 days to get the updated value? 

Can anyone help with this?

**Comment by elizafarley** [comment, marked as answer]
The 'xd' in metrics like `sales1d`, `sales7d`, etc. refers to the attribution window associated. So `sales1d` would return the value of sales that occurred during the 24 hours immediately following an ad click. `sales7d` would return the value of sales during the 7 days following an ad click. 

If you are using all available attribution windows (1, 7, 14, and 30 days), and requesting a report for date X you should request that date's report again on days X+1, X+7, X+14, and X+30. Due to the [restatement period](https://advertising.amazon.com/API/docs/en-us/reporting/overview#sponsored-ads), we also recommend pulling data on day X+60 to be sure you have the most up-to-date metrics. 

You should not need to sum these values as `sales30d` would contain all sales that occurred in the shorter attribution windows.
